packages/edit-post/src/editor.js

```diff
@@ -70,14 +71,24 @@ function Editor( {
 		[ currentPost.postType, currentPost.postId ]
 	);
 
+	// get the styles from the global styles
+	const { styles } = useStyles();
+
 	const editorSettings = useMemo(
 		() => ( {
 			...settings,
+			// TODO: This key should have a better name
+			__globalStyles: styles,
```

Comment:
It's not entirely clear to me why this is passed down from "edit-post" rather than just added automatically from `use-block-editor-settings.js` (directly in the editor package)

Reply:
And I agree that we need to find a better stable name here.

Reply:
> this is passed down from "edit-post" rather than just added automatically from use-block-editor-settings.js

There was no special reason 🙂 It seemed more natural to add it there, but I wasn't 100% sure about it to be honest. 

If we add it in `use-block-editor-settings.js` in the `editor` package, the same setting be available in both the post editor and the site editor, right?

Reply:
If we add it in use-block-editor-settings.js in the editor package, the same setting be available in both the post editor and the site editor, right?

yes

Reply:
makes sense. I moved it to the the `editor` package now 👍